boards/nxp/mr_mcxn_t1/mr_mcxn_t1_mcxn947_cpu1.dts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mcxn94x.dtsi>
+#include "mr_mcxn_t1.dtsi"
+
+/ {
+	model = "NXP FRDM_N94 board";
+	compatible = "nxp,mcxn947", "nxp,mcx";
+
+	cpus {
+		/delete-node/ cpu@0;
+	};
+
+	chosen {
+		zephyr,sram = &sramg;
+		zephyr,flash = &flash;
+		zephyr,flash-controller = &fmu;
+		zephyr,code-partition = &slot1_partition;
+		zephyr,console = &flexcomm2_lpuart2;
+		zephyr,shell-uart = &flexcomm2_lpuart2;
+	};
+};
+
+&flexcomm2 {
```

Comment:
Technically we don't support CPU1, so this can be quite confusing.
Let's minimize this config and remove all entries below except for (mbox, and edma)
Just keep sram, flash and code partition.

Reply:
Done